modules/create-a-kubeletconfig-crd-to-edit-kubelet-parameters.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-host-practices.adoc
+
+[id='create-a-kubeletconfig-crd-to-edit-kubelet-parameters-{context}']
+= Create a KubeletConfig CRD to edit kubelet parameters
+
+The kubelet configuration is currently serialized as an ignition configuration,
```

Comment:
Last I heard, we're not supporting the modification of ignition configs during installation. I'm not sure about the supportability of other use cases.

Reply:
I'm not sure if it's supported as far as Red Hat Support is concerned but it is something the user can do/change.

Reply:
OK! If QE hasn't tested it, we might need to remove this during review or conditionalize it for OKD.

Reply:
We're not really telling the user to edit it, it's just saying how it works now and that it can be edited.